libs/gi/ui/src/components/team/TeamCard.tsx

```
@@ -101,25 +117,47 @@ export function TeamCard({
             return (
-              <React.Fragment key={i}>
-                <Divider />
+              <Box
+                className="team-teammate"
+                sx={{
+                  border: '1px rgba(200,200,200,0.3) solid',
+                  '&:hover': {
+                    border: '1px rgba(200,200,200,0.8) solid',
+                  },
+                }}
+              >
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)